automation_tools/__init__.py

```diff
@@ -2027,27 +1979,71 @@ def product_install(distribution, certificate_url=None, selinux_mode=None, sat_v
 
     :param str distribution: product distribution wanted to install
     :param str certificate_url: where to fetch a fake certificate.
-    :param sat_version: Indicates which satellite version should be installed
+    :param str sat_version: Indicates which satellite version should be installed
 
     """
+    distribution = distribution.lower()
+    # Strip useless 'satellite6-' prefix if there is such
+    distribution = distribution[11:] if distribution.startswith('satellite6-') else distribution
     # Fetch the Satellite Version information.
     sat_version = sat_version or os.environ.get('SATELLITE_VERSION')
+    selinux_mode = selinux_mode or os.environ.get('SELINUX_MODE', 'enforcing')
```

Comment:
Why this and below are not converted to params? At admin_passwd is converted everywhere in this conversion PR.

Reply:
Applicable for such env vars in PR. Why ?

Reply:
The only place where I left ENV VARS is now `product_install()` - main task, from this task they are passed as params (and read from ENV VAR in subtask)

Tldr; this PR deals with using env vars in subtasks, for product_install I would raise another PR. Ok?